proposed/http-message.md

```diff
@@ -1100,7 +1106,8 @@ interface StreamableInterface
      *
      * @see seek()
      * @link http://www.php.net/manual/en/function.fseek.php
-     * @return bool Returns TRUE on success or FALSE on failure.
+     * @return bool Returns TRUE on success.
```

Comment:
I apologize, I don't think I was being clear. This method says that it will throw a RuntimeException on error and will return `true` on success, with false not allowed. My point is that returning any value at all is unnecessary because the failure case here is to throw an exception meaning that the return value is irrelevant. I also don't think we need to return the value returned by an underlying fseek() or rewind() call as those also have a true/false return value which we have abstracted into throwing an exception.